samples/csharp/MicrosoftEduGraphSamples/Workflows/ModuleWorkflow.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+using MicrosoftGraphSDK;
+using System.Resources;
+
+namespace MicrosoftEduGraphSamples.Workflows
+{
+    /// <summary>
+    /// Contains all the workflows related to Modules
+    /// </summary>
+    internal class ModuleWorkflow
+    {
+        private readonly IConfiguration _config;
+
+        public ModuleWorkflow(IConfiguration configuration)
+        {
+            this._config = configuration;
+
+            // Verify and throw exception for input values if null or empty
+            try
+            {
+                if (string.IsNullOrEmpty(_config["classId"]))
+                {
+                    throw new Exception("Missing classId please check appconfig.json file.");
+                }
+                else if (string.IsNullOrEmpty(_config["tenantId"]))
+                {
+                    throw new Exception("Missing tenantId please check appconfig.json file.");
+                }
+                else if (string.IsNullOrEmpty(_config["secret"]))
+                {
+                    throw new Exception("Missing secret please check appconfig.json file.");
+                }
+                else if (string.IsNullOrEmpty(_config["appId"]))
+                {
+                    throw new Exception("Missing appId please check appconfig.json file.");
+                }
+                else if (string.IsNullOrEmpty(_config["teacherAccount"]))
+                {
+                    throw new Exception("Missing teacherAccount please check appconfig.json file.");
+                }
+                else if (string.IsNullOrEmpty(_config["studentAccount"]))
+                {
+                    throw new Exception("Missing studentAccount please check appconfig.json file.");
+                }
+                else if (string.IsNullOrEmpty(_config["password"]))
+                {
+                    throw new Exception("Missing password please check appconfig.json file.");
+                }
+            }
+            catch (Exception ex)
+            {
+                throw;
+            }
+        }
+        /// <summary>
+        /// Workflow to create and publish the module
+        /// </summary>
+        public async Task ClassworkAsync()
+        {
+            try
+            {
+                // Get a Graph client using delegated permissions
+                var graphClient = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+
+                //create a draft module.
+                var module = await MicrosoftGraphSDK.Module.CreateAsync(graphClient, _config["classId"], "Module Workflow", "This is the first module created using Graph SDK.");
```

Comment:
change the Module displayName, generate a name based on date time instead of having "Module Workflow" hardcoded in the sample.

For instance:
**"Sample Module 09-05-2023 14:35:36"**

as we are doing [here](https://github.com/microsoft/edu-assignments-graph-sdk/blob/f71d3e63992b04d844e07b8a24d65014767bd484/samples/csharp/MicrosoftGraphSDK/Assignment.cs#L76C6-L76C6) when creating a new assignment

Reply:
Added extension date time along with module name